pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -266,6 +273,32 @@ type CiliumNetworkPolicyList struct {
 	Items []CiliumNetworkPolicy `json:"items"`
 }
 
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CiliumClusterwideNetworkPolicy is a Kubernetes third-party resource with an modified version
+// of CiliumNetworkPolicy which is cluster scoped rather than namespace scoped.
+type CiliumClusterwideNetworkPolicy struct {
+	CiliumNetworkPolicy
+
+	// Status is the status of the Cilium policy rule
```

Comment:
```golang
	// Status is the status of the Cilium policy rule
	// +optional
	// The reason this field exists in this structure is due a bug in the k8s code-generator
	// that doesn't create a `StatusUpdate` method because the field does not exist in
        // the structure.
```